### ISSUE TITLE
Potential inconsistency between Wiki links with no alt text and ones with

### ISSUE THREAD

**Issue by greyelf** [tweecode/twine#98]
I'm not sure if this is a bug or as designed in Sugarcane header.

When you create a wiki link with alternative text and the passage referenced does not exist then it assumes that the reference is actually a external link and is styled as if the link is valid.

If you click this link the browser with show a 'File Not Found' error (or words to that effect depending on browser) instead of the standard "Error: this passage does not exist" message.

eg.

```
[[Missing Passage]]  --> Shows in red and displays a "Error: this passage does not exist" message if clicked.
[[Alt Text|Missing Passage]] --> Shows as an external link and results in a 'File Not Found' error if clicked
```
